website/docs/reference/api/legacy/unleash/admin/features-v2.md

```diff
@@ -697,7 +705,15 @@ Content-Type: application/json; charset=utf-8
 
 ### PATCH variants for a feature toggle
 
-<ApiRequest verb="patch" url="api/admin/projects/:projectId/features/:featureName/variants" title="Patch variants for a feature toggle (example data)" payload={[{"op": "add", "path": "/1", "value": {
+:::caution
+
+From 4.21 variants are tied to an environment. Check our Open API docs for the up-to-date docs: https://docs.getunleash.io/reference/api/unleash/features
+
+ This endpoint affects all environments at once. If you only want to update a single environment, use [#update-variants-per-environment](#update-variants-per-environment) instead.
```

Comment:
So something like this? 

```suggestion
This API documentation is no longer maintained. You should use the [OpenAPI docs for this endpoint](/docs/reference/api/unleash/overwrite-feature-variants.api.mdx) instead.

This endpoint affects all environments at once. If you only want to update a single environment, use the [operation for updating variants per enviroment](/docs/reference/api/unleash/overwrite-feature-variants-on-environments.api.mdx) instead.
```

Reply:
Yes, but I can't find the linked doc, maybe it's autogenerated? You may know better than I do, so feel free to commit this suggestion because despite the link being correct is what I think we should say

Reply:
Yeah, the doc is generated, but you can generate it yourself. Anyway, this should link to the right place.